src/azul/__init__.py

```diff
@@ -1485,6 +1485,8 @@ def docker_images(self) -> dict[str, ImageSpec]:
 
     waf_rate_rule_retry_after = 30  # seconds
 
+    waf_rate_rule_limit = 1000  # limit must be between 100 and 20,000,000
```

Comment:
```suggestion
    waf_rate_rule_limit = 1000
    
    assert 100 <= waf_rate_rule_limit <= 2_000_000_000 # mandated by AWS
```

https://docs.aws.amazon.com/waf/latest/APIReference/API_RateBasedStatement.html#WAF-Type-RateBasedStatement-Limit